<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(50, 75, 77);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,70,0,30">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>MS UI Gothic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color:rgb(255, 255, 255);

</string>
      </property>
      <property name="text">
       <string>Analizador de Espectro</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GraphWidget" name="spectrumGraph" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(50, 75, 77);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>8</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
      </property>
      <property name="text">
       <string>Control de Se√±ales</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="signalList">
        <property name="styleSheet">
         <string notr="true">font: 63 7pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="removeSignal">
          <property name="styleSheet">
           <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeAllSignals">
          <property name="styleSheet">
           <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
          </property>
          <property name="text">
           <string>Eliminar todo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toggleSignal">
          <property name="styleSheet">
           <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
          </property>
          <property name="text">
           <string>Toggle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">GUI/FrontEnd/graphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
